lib/rucio/rse/rsemanager.py

```diff
@@ -169,7 +169,7 @@ def select_protocol(rse_settings, operation, scheme=None, domain='wan'):
     return min(candidates, key=lambda k: k['domains'][domain][operation])
 
 
-def create_protocol(rse_settings, operation, scheme=None, domain='wan', auth_token=None, logger=logging.log):
+def create_protocol(rse_settings, operation, scheme=None, domain='wan', auth_token=None, logger=logging.log, protocol_attr=None):
```

Comment:
`logger` should be the last one, add `protocol_attr` before. Also describe it below in the doc.

Reply:
It's done!